lib/shopify_api/utils.rb

```diff
@@ -0,0 +1,22 @@
+# typed: strict
+# frozen_string_literal: true
+
+module ShopifyAPI
+  module Utils
+    extend T::Sig
+
+    sig do
+      params(
+        fields: T.untyped
+      ).returns(T.nilable(ShopifyAPI::Shop))
+    end
+    def self.current_shop(fields: nil)
```

Comment:
We could also have the session be passed in as an optional argument to these, so we're not restricted to using `activate_session` only.